backend/clickhouse/query.go

```diff
@@ -338,7 +344,7 @@ func KeysAggregated(ctx context.Context, client *Client, tableName string, proje
 	return keys, rows.Err()
 }
 
-func KeyValuesAggregated(ctx context.Context, client *Client, tableName string, projectID int, keyName string, startDate time.Time, endDate time.Time, query *string, limit *int) ([]string, error) {
+func KeyValuesAggregated(ctx context.Context, client *Client, tableName string, projectID int, keyName string, startDate time.Time, endDate time.Time, query *string, limit *int, additionalWhereClauses *[]string) ([]string, error) {
```

Comment:
Opted to pass in the where clauses as a string since `event` is specific to one search. However, may be better to pass in another way?

Reply:
Updated to pass in the `event` into `KeysAggregated` and handle it in there with `sb.Var` to avoid sql injection converns